packages/docs/content/5_components/accordion.mdx

```diff
@@ -8,11 +8,29 @@ An accordion is a list of headers that hide or reveal additional content when se
 
 ## `<Accordion>`
 
+<StorybookExample
+  componentName="accordion"
+  sourceFilePath="components/Accordion/Accordion.tsx"
+  storyId="components-accordion--default"
+/>
```

Comment:
The old page's static HTML example was pretty useless. I'm glad that's not in here. When I interact with this new one and try to open the second panel without closing the first, it gets cut off, but that would be hard to avoid. That's out of scope, but I was pondering what a solution might look like in the future